### PR TITLE
Lab extension module 1 - 3

### PR DESCRIPTION
**Pull request by bendesu**
drodown menu in notebook:
dropdown_menu

button appearance in lab:
lab-button

module2(parameters dialog) in lab:
lab-params

### CODE REVIEW

**Comment by bendesu** [comment]
dropmenu
API Key
committed

**Comment by bendesu** [comment]
settings

**Comment by aakashns** [comment]
Please use the `npm` package manager, not yarn

File: jovian/jovian_lab_ext/LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
+                           Version 2.0, January 2004
```
Comment (aakashns) [inline jovian/jovian_lab_ext/LICENSE]: we don't need a licence file. Please remove this.

File: jovian/jovian_lab_ext/tsconfig.tsbuildinfo [hunk 2]
```diff
@@ -0,0 +1,4519 @@
+{
+  "program": {
```
Comment (aakashns) [inline jovian/jovian_lab_ext/tsconfig.tsbuildinfo]: please ignore this file in `.gitignore`

**Comment by aakashns** [comment]
Please rename the folder and the package name to `jovian-jupyter-lab-extension`

File: jovian/jovian_lab_ext/tsconfig.tsbuildinfo (hunk 2 above)
Comment (bendesu) [reply jovian/jovian_lab_ext/tsconfig.tsbuildinfo]: Do I add **.gitignore** to **jovian/jovian_lab_ext/tsconfig.tsbuildinfo**?

**Comment by bendesu** [comment]
Finish the final change on module 1-3 for Lab Extension

**Comment by bendesu** [comment]
UI

New UI to match new params

**Pull request closed without merging**